CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # did-io ChangeLog
 
+## 0.8.0
+(Migrating from v0.7.0 will require renaming the relevant properties in existing
+DID Docs.)
+
+## Changed
+
+- (BREAKING) Updated terminology to match latest VeresOne DID method specs:
+  `grantCapability -> capabilityDelegation`,
+  `invokeCapability -> capabilityInvocation`
+- (BREAKING) Renamed `privateKey` to `secretKey` properties, to match Digital
+  Bazaar conventions.
```

Comment:
The PR title and this are written backwards.  It's also functions.  Perhaps:
```Renamed `secretKey` to `privateKey`, to match Digital Bazaar conventions.```

Reply:
Whoops, thanks!